eclass/kernel-install.eclass

```diff
@@ -386,7 +530,7 @@ kernel-install_pkg_pretend() {
 		ewarn "to pause or abort the build process and install it before"
 		ewarn "resuming."
 
-		if use initramfs; then
+		if use initramfs && ( ! in_iuse generic-uki || ! use generic-uki ); then
```

Comment:
Unnecessary subshell, and unnecessary `in_iuse` since we know exactly when it's going to be present.

Reply:
> Unnecessary subshell, and unnecessary `in_iuse` since we know exactly when it's going to be present.

To me it looks a bit weird without the `( )` . Since we want to output this message if we have use initiramfs and no `KERNEL_IUSE_GENERIC_UKI` or if we have use initramfs and use generic-uki is disabled.

Reply:
Use `{ ...; }`.

Reply:
Fixed